apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -347,6 +367,8 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     */
     function canClose(uint256 _voteId) external view returns (bool) {
         Vote storage vote_ = _getVote(_voteId);
+        // Note: do we check for isExecuted() because the original call to close may fail (but not revert) for some reason?
+        // Note: wondering if we should expose a permissionless method to close any cancelled vote (and that way we can also set their status to be cancelled)?
         return (_isActive(vote_) || _isExecuted(vote_)) && _hasEnded(vote_);
```

Comment:
This is also affected by above—submitters wouldn't be able to manually close the vote from the Agreement's side.

Reply:
> do we check for isExecuted() because the original call to close may fail (but not revert) for some reason?

I'm having a hard time trying to remember why I added this, but I think this was introduced before we added the close action step in `executeVote`, now it seems redundant

> wondering if we should expose a permissionless method to close any cancelled vote (and that way we can also set their status to be cancelled)?

About adding an extra method to allow closing canceled votes, I wouldn't add it to be honest. Users can go through the agreement app instead. I wouldn't add unnecessary methods to the interface cause we are already close to the limit.

Reply:
Addressed in f7a96d8b837a4e03b934a5115638b2a411db2aad